internal/cloud/realm/app.go

```diff
@@ -0,0 +1,74 @@
+package realm
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/10gen/realm-cli/internal/utils/api"
+)
+
+// App represents basic Realm App data
+type App struct {
+	ID          string `json:"_id"`
+	GroupID     string `json:"group_id"`
+	ClientAppID string `json:"client_app_id"`
+	Name        string `json:"name"`
+}
+
+var (
+	errGroupNotFound = errors.New("group could not be found")
```

Comment:
for now let's just inline this error, I'm not sure where else we'd be using it yet